Assets/Scripts/Replay/ReplayManager.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Common;
+using Inputs;
+using BTManager;
+
+namespace Replay
+{
+    public struct QueueData
+    {
+        public int frameNo;
+        public List<InputAction> action;
+        public int playerID;
+    }
+
+    public class ReplayManager : Singleton<ReplayManager>
+    {
+
+        public Queue<QueueData> savedQueueData;
```

Comment:
why are there 2 queue data structures?

Reply:
One for gameplay and another for replay mode